build_defs.bzl

```diff
@@ -55,6 +64,20 @@ def pybind_extension(
         **kwargs
     )
 
+    copy_file(
+        name = name + "_copy_file",
```

Comment:
IIUC this name isn't referenced from anywhere, but I'd still want to make it more unique (instead of reusing the name of the rule), maybe `_copy_so_pyd`?

That way it's easier to refer to this in comments and chats.

Reply:
Sure, changed to `name + "_copy_so_to_pyd"`.